webpack/ForemanTasks/Components/TasksTable/Components/ConfirmModal/ConfirmModal.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { startCase } from 'lodash';
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import ForemanModal from 'foremanReact/components/ForemanModal';
+import { useForemanModal } from 'foremanReact/components/ForemanModal/ForemanModalHooks';
+import { CANCEL_MODAL, RESUME_MODAL } from '../../TasksTableConstants';
+
+export const ConfirmModal = ({
+  actionText,
+  actionState,
+  tasksActions,
+  actionType,
+  selectedRowsLen,
+  id,
+}) => {
+  const { setModalClosed } = useForemanModal({
+    id,
+  });
+  if ([CANCEL_MODAL, RESUME_MODAL].includes(actionType)) {
+    selectedRowsLen = 1;
```

Comment:
it's not a good practice to mutate the props,
can this logic be part of [TasksTablePage selectedRowsLen const](https://github.com/theforeman/foreman-tasks/pull/538/files#diff-8de9b4e575996966ab1cae4c0ed76713R107)

Reply:
The issue is that tasksTablePage doesn't have the `actionType`. But I'll change this logic in confirmModal